### PR TITLE
houdini: 20.0.688 -> 20.5.278

### PR DESCRIPTION
**Pull request by permahorse**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by pedohorse** [comment]
1 package built:
houdini

Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>houdini</li>
  </ul>
</details>

File: pkgs/applications/misc/houdini/runtime.nix [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 
 callPackage ./runtime-build.nix rec {
-  version = "20.0.688";
+  version = "20.5.278";
   eulaDate = "2021-10-13";
-  src = requireFile rec {
+  src = requireFile {
     name = "houdini-${version}-linux_x86_64_gcc11.2.tar.gz";
-    sha256 = "99f9088824c328de9d351f037f26ff1cba960fbd9b4e2ed1d52601680d3512a6";
+    sha256 = "3b9a00a220e2bbec6b9837d39e294fa99b9ef3e7575531e72b4cab5fa9a84d27";
```
Comment (gador) [inline pkgs/applications/misc/houdini/runtime.nix:8]:
```suggestion
    hash = "3b9a00a220e2bbec6b9837d39e294fa99b9ef3e7575531e72b4cab5fa9a84d27";
```

**Review by gador — CHANGES_REQUESTED** [review]

**Comment by gador** [comment]
Also please rebase to resolve merge conflicts

File: pkgs/applications/misc/houdini/runtime.nix (hunk 1 above)
Comment (pedohorse) [reply pkgs/applications/misc/houdini/runtime.nix:8]: doesn't this require appending `sha256-` to the hash itself?

Comment (pedohorse) [reply pkgs/applications/misc/houdini/runtime.nix]: ok, done, had to convert hash to sri form

Comment (gador) [reply pkgs/applications/misc/houdini/runtime.nix]: yes. and converting it. It should be `sha256-O5oAoiDiu+xrmDfTnilPqZue8+dXVTHnK0yrX6moTSc=` (got with `nix-hash --to-sri --type sha256 3b9a00a220e2bbec6b9837d39e294fa99b9ef3e7575531e72b4cab5fa9a84d27`).
I don't have `houdini` locally so you can get the hash with `nix-prefech-url`

**Review by gador — APPROVED** [review]
LGTM!

**Pull request merged**